twitch4j/src/main/java/com/github/twitch4j/TwitchClientBuilder.java

```diff
@@ -137,6 +139,12 @@
     @With
     private EventManager eventManager = null;
 
+    /**
+     * EventManager
+     */
+    @With
+    private Class<? extends IEventHandler> defaultEventHandler = null;
```

Comment:
Perhaps this could be replicated in the module-specific builders as well

Reply:
I added it to all builders that have a eventManager reference

Reply:
LGTM & good doc changes 👍